web-external/src/style/full-viewport.styl

```diff
@@ -1,24 +1,23 @@
-
 html
-  height: 100%
+  height 100%
 
 body
-  position: absolute
-  top: 0
-  bottom: 0
-  left: 0
-  right: 0
+  position absolute
+  top 0
+  bottom 0
+  left 0
+  right 0
 
 #g-global-nav-container
-  z-index: 1
+  z-index 1
 
-#g-app-footer-container
-  position: fixed
-  bottom: 0
-  left: 0
-  right: 0
+DISABLE#g-app-footer-container
+  position fixed
+  bottom 0
+  left 0
+  right 0
 
-#g-app-body-container
-  height: 95%
-  padding-bottom: 88px
+DISABLE#g-app-body-container
+  height 95%
+  padding-bottom 88px
```

Comment:
That will work, but we have an @media CSS rule that overrides the app-body-container style you had added (so the margin was reset).

As a design choice, I prefer the footer that doesn't stick to the bottom of the window, as it eats up space where we can be displaying data, and nothing we have in the footer is a common UI action)